.gitignore

```diff
@@ -151,6 +151,8 @@ history.txt
 
 arc-script/**/template/
 
+*.crate
+
```

Comment:
If you happen to create a crate somewhere in the source tree in a directory where Lit looks for tests, you will be spammed with a lot of inconclusive tests. In that case it's easier to remove them by just asking Git for what fouled the source tree.